src/server/server_family.cc

```diff
@@ -825,6 +826,40 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
                             MetricType::COUNTER, &resp->body());
   AppendMetricWithoutLabels("net_output_bytes_total", "", conn_stats.io_write_bytes,
                             MetricType::COUNTER, &resp->body());
+  {
+    string send_latency_metrics;
+    constexpr string_view kReplyLatency = "reply_latency_usec_total";
+    AppendMetricHeader(kReplyLatency, "Reply latency per mode", MetricType::COUNTER,
+                       &send_latency_metrics);
+
+    string send_count_metrics;
+    constexpr string_view kReplyCount = "reply_count";
```

Comment:
I think the naming convention of both metrics not according to the prometheus standard. See https://prometheus.io/docs/practices/naming/

specifically, latency must be seconds and must be end with "duration_seconds" imho. You can also see how prometheus itself names its metrics by starting our monitoring stack and checking out `:9090/metrics`

Reply:
That's a good resource, thanks for sharing!
(done)